analyzers/tests/SonarAnalyzer.UnitTest/Rules/ObsoleteAttributesTest.cs

```diff
@@ -24,25 +24,49 @@
 namespace SonarAnalyzer.UnitTest.Rules;
 
 [TestClass]
-public class RemoveObsoleteCodeTest
+public class ObsoleteAttributesTest
 {
-    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.RemoveObsoleteCode>();
-    private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.RemoveObsoleteCode>();
+    private readonly VerifierBuilder builderExplanationNeededCS =
```

Comment:
The "builder" part is not needed here. It only makes sense when there is nothing else to add as a name. It can be removed.